src/predicates/isAlt.js

```diff
@@ -1,5 +1,6 @@
 /** @license ISC License (c) copyright 2017 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 
-module.exports =
-  require('../core/isAlt')
+import isAlt from '../core/isAlt.js'
+
+export default isAlt
```

Comment:
could these be merged? export { default as isAlt } from '../core/isAlt'? all the ones below too

Reply:
I'm ok, with above proviso.